quinn-proto/src/connection.rs

```diff
@@ -630,11 +642,13 @@ impl Connection {
             }
         }
         if lost_ack_eliciting {
-            self.congestion_event(now, largest_lost_time)
+            if let Some(lost_time) = largest_lost_time {
+                self.congestion_event(now, lost_time);
+            }
```

Comment:
`largest_lost_time` should be unwrapped rather than guarded here, because it is always set if lost_ack_eliciting is set (although the reverse does not hold).

Reply:
Fixed.